features/test/new_api.feature

```diff
@@ -27,3 +27,8 @@ Feature: test new api methods
     Given I switch to cluster admin pseudo user
     And I use the "openshift-monitoring" project
     And evaluation of `route('prometheus-k8s').spec.host` is stored in the clipboard
+
+  @admin
+  Scenario: check cluster_version capabilities
+    And the expression should be true> cluster_version('version').capability_is_enabled?(capability: 'ImageRegistry')
```

Comment:
@pruan-rht Is skip this test if `ImageRegistry` capabilities are not enabled to avoid failure.

Reply:
@dpuniaredhat this is just an example of how to use it.  Once this PR is merged, then you can update your PR to use `cluster_version('version').capability_is_enabled?(capability: 'ImageRegistry')` in your step_definition 
```
unless is_capability_enabled?(capability)
```

Reply:
Thanks, Got it